README.md

```diff
@@ -29,7 +29,14 @@ datastreams.Start(datastreams.WithAgentAddr("notlocalhost:8126"))
 ```
 
 The instrumentation relies on creating checkpoints at various points in your data stream services, recording the pathway that messages take along the way. These pathways are stored within a Go Context, and are passed around via message headers.
- 
+
+### M1 support
+
+On M1 machines, the following parameters are needed when starting the instrumented service:
```

Comment:
Just a little bit of context: This library relies on [Confluent Kafka Go](https://github.com/confluentinc/confluent-kafka-go)

Reply:
done